src/components/FeatureSelector/FeatureMenu.tsx

```diff
@@ -1,35 +1,86 @@
 
+const useStyles = makeStyles(theme => ({
+  popover: {
+    pointerEvents: 'none'
+  },
+  paper: {
+    padding: theme.spacing(1)
+  },
+  button: {
+    margin: theme.spacing(1),
+    whiteSpace: "nowrap"
+  },
+  root: {
+    width: '100%',
+    minWidth: 200
+  }
+}))
+
 export default function FeatureMenu(props: {
     onSelect: (option: FeatureSliderData) => void
     hidden: string[]
   }) {
-    
+    const classes = useStyles()
     const [anchorEl, setAnchorEl] = React.useState(null);
     const handleClose = () => {
       setAnchorEl(null);
     };
     const handleMenuOpen = (event: any) => {
       setAnchorEl(event.currentTarget);
     };
+    const [popupAnchor, setPopupAnchor] = useState(null)
     
     return (
-      <div>
-        <Button aria-controls="feature-menu" aria-haspopup="true" onClick={handleMenuOpen}>
-          Add Feature Filter
-        </Button>
-        <Menu
-          id="feature-menu"
-          anchorEl={anchorEl}
-          keepMounted
-          open={!!anchorEl}
-          onClose={handleClose}
-        >
-          {Options.filter(o => !props.hidden.includes(o.name)).map((o, index) => (<MenuItem key={index} onClick={() => props.onSelect(o)}>{o.name}</MenuItem>))}
-        </Menu>
+      <div className = {classes.root}>
+        <List component={Paper} disablePadding={true}>
+            <ListItem style={{justifyContent:"space-between"}}> 
+              <Button className={classes.button}  aria-controls="feature-menu" aria-haspopup="true" onClick={handleMenuOpen} >
+                Add Feature Filter
+              </Button>
+              <Menu
+                id="feature-menu"
+                anchorEl={anchorEl}
+                keepMounted
+                open={!!anchorEl}
+                onClose={handleClose}
+              >
+                {Options.filter(o => !props.hidden.includes(o.name)).map((o, index) => (<MenuItem key={index} onClick={() => props.onSelect(o)}>{o.name}</MenuItem>))}
+              </Menu>
+
+              <Box style={{ padding: 12 }}>
+                  <InfoIcon
+                    onMouseEnter={(event: any) => setPopupAnchor(event.currentTarget)}
+                    onMouseLeave={() => setPopupAnchor(null)}
+                  />
+                </Box>
+              <Popover
+                  id='mouse-over-popover'
+                  className={classes.popover}
+                  classes={{
+                    paper: classes.paper
+                  }}
+                  open={!!popupAnchor}
+                  anchorEl={popupAnchor}
+                  anchorOrigin={{
+                    vertical: 'top',
+                    horizontal: 'center'
+                  }}
+                  transformOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'center'
+                  }}
+                  disableRestoreFocus
+                 >
+                {popupAnchor == null ? "" : "These features make up a song"}
```

Comment:
This text doesn't really tell the user much...

Maybe this popup should be on the text in each slider. Give a one liner as to what each feature means there

Also you don't want to do this `popupAnchor == null` check. It makes the popover look weird when it's closing

Reply:
removed check and changed location to audioFeatureSlider